python/pyspark/context.py

```diff
@@ -230,6 +230,14 @@ def _do_init(self, master, appName, sparkHome, pyFiles, environment, batchSize,
         self.pythonExec = os.environ.get("PYSPARK_PYTHON", 'python3')
         self.pythonVer = "%d.%d" % sys.version_info[:2]
 
+        if self.pythonVer == (3, 6):
```

Comment:
nit: I guess `self.pythonVer < (3, 7)` is better?

Reply:
Cool, can we still warn `Python 3.6` only in the message?

Reply:
Yes, that's fine.

Reply:
ah, sorry, it should be `sys.version_info < (3, 7)`.